src/main/java/it/smartcommunitylab/aac/webauthn/model/WebAuthnLoginResponse.java

```diff
@@ -0,0 +1,64 @@
+package it.smartcommunitylab.aac.webauthn.model;
+
+import java.io.IOException;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.yubico.webauthn.AssertionRequest;
+
+@Valid
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class WebAuthnLoginResponse {
+
+    @JsonProperty("key")
+    @NotNull
+    private String key;
+
+    @JsonProperty("assertionRequest")
+    @JsonSerialize(using = AssertionRequestSerializer.class)
+    @NotNull
+    AssertionRequest assertionrequest;
```

Comment:
visibility private

Reply:
Fixed in 8f9b6b7b3b42efaaec39d5aa8cbcd60f5e223ea5